Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/eventregistration/attendee/checkins/CheckInsFragment.kt

```diff
@@ -42,11 +48,15 @@ class CheckInsFragment : Fragment(R.layout.trace_location_attendee_checkins_frag
         factoryProducer = { viewModelFactory },
         constructorCall = { factory, savedState ->
             factory as CheckInsViewModel.Factory
-            factory.create(savedState = savedState, deepLink = navArgs.uri)
+            factory.create(
```

Comment:
Why did you refactor this?

Reply:
What refactoring ? You mean the formatting?

Reply:
Yes, seemed very random :grin: